app/views/spree/admin/subscriptions/index.html.erb

```diff
@@ -87,12 +87,12 @@
         </div>
         <div class="field-block col-3">
           <div class="field">
-            <%= label_tag :q_with_line_item_eq, SolidusSubscriptions::Subscription.human_attribute_name(:with_line_item) %>
+            <%= label_tag :q_with_subscribable_eq, SolidusSubscriptions::Subscription.human_attribute_name(:with_subscribable) %>
 
             <%=
               f.select(
-                :with_line_item,
-                options_for_select(::SolidusSubscriptions::LineItem.joins(:subscribable).map { |item| [ item.subscribable.name, item.id ]}, params.dig(:q, :with_line_item)),
+                :with_subscribable,
```

Comment:
More than a select field, this calls for a search field that does an API call to avoid the query altogether when loading the page. WDYT?

Reply:
Sorry I've seen this too late. I think this would be an even better UX, especially in stores with lots of subscribable variants. I'll open a ticket for it.

Reply:
That's a great idea. I merged this as is because this fixes a bug, but this improvement would be great.